examples/python_client.md

```diff
@@ -0,0 +1,21 @@
+#Python Client Examples
+
+```
+from perch-client import PerchAPIClient
+
+perch = PerchAPIClient(username='username@example.com', password='your-perch-password', api_key='your-api-key')
+
+perch.me.get()
```

Comment:
If the CRUD class allowed `endpoint.get()` to be called using `endpoint()`, this could be written as `perch.me()`. It flies in the face of "only one obvious way to do it", but it's also pretty :P